src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBody.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.legaladvisor;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDHearingDurationType;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDOrderDirectionType;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDDirectionPartyType;
+import uk.gov.hmcts.cmc.claimstore.idam.models.UserDetails;
+import uk.gov.hmcts.reform.docassembly.domain.FormPayload;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Builder
+@EqualsAndHashCode
+public class DocAssemblyTemplateBody implements FormPayload {
+
+    private static final String HMCTS_URL = "https://www.gov.uk/make-money-claim";
```

Comment:
private constants required? can we just assign the value directly to courtSeal?

Reply:
we can - the HMCTS_URL is used in multiple places so I will leave it as a constant, but I can inline the courtseal

Reply:
if its constant value, better to have a separate file so can be used. Having constants in Body class seems wrong to me. Just thought. Its up to you